useFetch

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+
+export default function useFetch(url) {
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    if (url) {
+      fetch(url)
+        .then(response => response.json())
+        .then(data => {
+          setData(data);
+        })
+        .catch(error_ => {
+          setError(error_);
```

Comment:
hier könntest du noch ein `console.error`mit unterbringen, damit du den Error nicht nur ablegst, sondern zumindest auch in der Konsole ausgibst.